### PR TITLE
Climate: hide "Currently: °C" in case the thermostat doesn't report the current temperature

### PR DESCRIPTION
**Pull request by tkintscher**
This code checks if the thermostat reports the current room temperature.
If so, it is printed next to the target temperature (this is the current behavior).
If no valid number is returned, only the target temperature is shown, and the (unavailable) room temperature is hidden.

Fixes: https://github.com/home-assistant/home-assistant-polymer/issues/654
Fixes: https://github.com/home-assistant/home-assistant/issues/3947

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @tkintscher, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant-polymer#655).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by balloob** [comment]
In what situation will a climate entity not have a current temperature?

**Comment by bieniu** [comment]
Most Z-Wave radiator thermostats don't report current_temperature to the controller eg Danfoss 014G0013. There ara a lot posts about that in [community](https://community.home-assistant.io/t/danfoss-lc-13-currently-empty/25959). IMO the ideal solution would be to get the temperature value from another sensor as in the generic_thermostat component.

**Comment by tkintscher** [comment]
Yes, using another sensor as input for the current temperature would be useful as well.
However, I don't have separate temperature sensors in all rooms and in this case, hiding the (unavailable) current temperature is an improvement.

Reading the temperature from another sensor is a separate issue as it is not just a frontend fix, but requires changes to the zwave module.

File: src/state-summary/state-card-climate.html [hunk 1]
```diff
@@ -43,8 +43,7 @@
             [[computeTargetTemperature(stateObj)]]
           </span>
         </div>
-
-        <div class='current'>
+        <div class='current' hidden$='[[!hasCurrentTemperature(stateObj)]]'>
```
Comment (balloob) [inline src/state-summary/state-card-climate.html]: No reason to add a function. You can just pass in `stateObj.attributes.current_temperature`. If the temperature cannot be rendered, we should still show it.

**Comment by NovapaX** [comment]
I'm going to close this here, further discussion (if needed) in #1047

**Pull request closed without merging**